Docs/developer_guide/build_instructions/macos.md

```diff
@@ -102,16 +140,43 @@ See list of issues common to all operating systems on [Common errors](common_err
 ### error while configuring PCRE: "cannot run C compiled program"
 
 If the XCode command line tools are not properly set up on macOS, PCRE could fail to build in the Superbuild process with the errors like below:
-```
+
+```bash
 configure: error: in `/Users/fedorov/local/Slicer4-Debug/PCRE-build':
 configure: error: cannot run C compiled programs.
 ```
 
 To install XCode command line tools, use the following command from the terminal:
-```
+
+```bash
 xcode-select --install
 ```
 
 ### dyld: malformed mach-o: load commands size (...) > 32768
 
-Path of source or build folder is too long. For example building Slicer in */User/somebody/projects/something/dev/slicer/slicer-qt5-rel* may fail with malformed mach-o error, while it succeeds in */sq5* folder. To resolve this error, move source and binary files into a folder with shorter full path and restart the build from scratch (the build tree is not relocatable).
+Path of source or build folder is too long. For example building Slicer in */User/somebody/projects/something/dev/slicer/slicer-qt5-rel* may fail with malformed mach-o error, while it succeeds in */opt/s* folder. To resolve this error, move source and binary files into a folder with shorter full path and restart the build from scratch (the build tree is not relocatable).
+
```

Comment:
We could clarify here that only the build tree needs to be in a short path.  Source can be anywhere.

Reply:
On Windows path length of source and build tree is enforced to 40 characters. Does this need to be changed and also enforced on other platforms? [PreventDirWithTooManyChars.cmake](https://github.com/Slicer/Slicer/blob/767551b4b8cf85f9d3193af1a5ff4ff8f3fdf728/CMake/PreventDirWithTooManyChars.cmake)

https://github.com/Slicer/Slicer/blob/767551b4b8cf85f9d3193af1a5ff4ff8f3fdf728/CMakeLists.txt#L85-L92

Reply:
I believe that @jcfr said that the new vtk will have a lot fewer python related shared libraries so we can test if that removes this as a practical concern on mac. But yes, a note on the windows build instructions would make sense.

Reply:
Ditto. With VTK9, we won't have the PythonD library anymore.

Reply:
We already have a note in Windows build instructions on using short paths: "Due to maximum path length limitations during build the build process, source and build folders must be located in a folder with very short (couple of characters) total path length."  (https://slicer.readthedocs.io/en/latest/developer_guide/build_instructions/windows.html#set-up-source-and-build-folders)